pxr/usdImaging/usdImaging/lightAdapter.cpp

```diff
@@ -177,6 +216,9 @@ UsdImagingLightAdapter::ProcessPropertyChange(UsdPrim const& prim,
     if (UsdGeomXformable::IsTransformationAffectedByAttrNamed(propertyName)) {
         return HdLight::DirtyBits::DirtyTransform;
     }
+
+    _UpdateCollectionsChanged(prim, cachePath);
```

Comment:
We'd want to guard this in "if (propertyName == lightLinkAttrToken)" or something, not sure what the right token is offhand.

Reply:
Indeed! Would this be OK? Some string parsing as the attributes can be numerous and are all under the collection:lightLink/shadowLink prefix:
```
    if (TfStringStartsWith(propertyName.GetString(), UsdImagingTokens->collectionShadowLink.GetString()) || 
        TfStringStartsWith(propertyName.GetString(), UsdImagingTokens->collectionLightLink.GetString()))
    {
        _UpdateCollectionsChanged(prim, cachePath);
    }
```

Reply:
Yeah, that would be great!